roles/node_prep/tasks/40_bridge.yml

```
@@ -67,7 +67,7 @@
         state: present
       when: (not enable_virtualmedia) and
             ((ipv4_provisioning|bool) or (not ipv6_enabled|bool) or
-            ((release_version.split('.')[0]|int == 4) and (release_version.split('.')[1]|int == 3)))
+            (release_version is ansible.builtin.version('4.3', '=='))
```

I think here we are missing a `)` to close the or expression that started in the line above. 
```suggestion
            (release_version is ansible.builtin.version('4.3', '==')))
```
https://www.distributed-ci.io/jobs/07318323-753a-4add-b0ed-7295b7183cef/jobStates?sort=date&task=855b07b5-2f23-441f-b7ac-0e8f6615be48

---

True, changed